docs/build-docs.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+rm -f ./_FIRST_RUN.flag
+
+export PATH="${CONDA}/bin:${PATH}"
+
+curl \
+    -L \
+    -o ${HOME}/miniforge.sh \
+    https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh
+
+/bin/bash ${HOME}/miniforge.sh -b -p ${CONDA}
+
+conda config --set always_yes yes --set changeps1 no
```

Comment:
Maybe we need to update `conda` itself here?
https://github.com/microsoft/LightGBM/blob/d31346f6c265e994c54ebddd19ed1fb6445ba0c9/.ci/setup.sh#L115

Reply:
sure

Reply:
updated in https://github.com/microsoft/LightGBM/pull/4957/commits/8010f99fda097fa9702b9c900f31d5d078eaf1fc